webscraper.js

```diff
@@ -35,12 +39,42 @@ if (require.main === module) {
                                             };
         }
 
-        return doc_dimensions;
-      }).then(function(dimensions) {
+        setTimeout(function () {
+           callback({
+              html: src,
+              css: node_properties,
+              folder : dir
+           });
+        });
+
+      }, item).then(function(doc_info) {
```

Comment:
Are you sure? I'm making a few guesses about Selenium API as I read it, but I suspect the `then()` receives the argument you pass to `callback`, not `item`.